lib/localization/locales/sv.json

```
@@ -0,0 +1,64 @@
+{
+  "loading": "Laddar...",
+  "why_am_i_seeing": "Varför ser jag detta?",
+  "protected_by": "Skyddat av",
+  "protected_from": "Från",
+  "made_with": "Gjort med ❤️ i 🇨🇦",
+  "mascot_design": "Maskotdesign av",
+  "ai_companies_explanation": "Du ser detta eftersom att administratören av denna webbsida har upprättat Anubis-systemet för att skydda servern mot plågan av att AI-företag aggressivt skrapar webbsidor. Detta kan orsaka driftstopp för webbsidor, vilket gör deras resurser otillgängliga för alla.",
+  "anubis_compromise": "Anubis är en kompromiss. Anubis använder sig av ett arbetsbevissystem på samma sätt som Hashcash, ett förslag om arbetsbevissystem för att minska epostspam. Idén är att den extra belastningen är obetydlig på en individuell skala, men att den på massskrapningsnivåer adderas upp och gör processen mycket dyrare.",
+  "hack_purpose": "I slutändan är detta ett hack vars funktion är att ge en \"tillräckligt bra\" lösning så att mer tid kan spenderas på att identifiera headless-webbläsare (t.ex. via hur dem hanterar typsnittsrendering) så att arbetsbevissidan inte måste presenteras för användare som är mycket mer troliga att vara riktiga.",
+  "jshelter_note": "Notera att Anubis kräver användningen av moderna JavaScript-funktioner som tillägg såsom JShelter kommer att avaktivera. Var vänlig och avaktivera JShelter eller andra liknande tillägg för denna domän.",
+  "version_info": "Den här webbsidan kör Anubis version",
+  "try_again": "Försök igen",
+  "go_home": "Gå hem",
+  "contact_webmaster": "eller om du tycker att du inte borde bli blockerad, kontakta den webbansvarige på",
+  "connection_security": "Var vänlig och vänta en stund medan vi säkerställer din anslutnings säkerhet.",
+  "javascript_required": "Tyvärr måste du slå igång JavaScript för att komma förbi denna utmaning. Detta eftersom AI-företag har ändrat samhällskontraktet gällande webbhosting. En lösning som icke kräver JavaScript ett pågående arbete.",
+  "benchmark_requires_js": "För att köra prestandamätningsverktyget krävs det att JavaScript är igång.",
+  "difficulty": "Svårighetsgrad:",
+  "algorithm": "Algoritm:",
+  "compare": "Jämför:",
+  "time": "Tid",
+  "iters": "Iterationer",
+  "time_a": "Tid A",
+  "iters_a": "Iterationer A",
+  "time_b": "Tid B",
+  "iters_b": "Iterationer B",
+  "static_check_endpoint": "Detta är bara en kontrollendpunkt för andvänding av din reverse-proxy.",
+  "authorization_required": "Tillstånd krävs",
+  "cookies_disabled": "Din webbläsare är konfigurerad för att inaktivera cookies. Anubis kräver cookies för att säkerställa att du är en giltig klient. Var vänlig och aktivera cookies för den här domänen.",
```

Does not have ending `.` in english locale.
```suggestion
  "cookies_disabled": "Din webbläsare är konfigurerad för att inaktivera cookies. Anubis kräver cookies för att säkerställa att du är en giltig klient. Var vänlig och aktivera cookies för den här domänen",
```